packages/jest-circus/src/legacy-code-todo-rewrite/jestExpect.ts

```diff
@@ -15,18 +15,18 @@ import {
   toThrowErrorMatchingSnapshot,
 } from 'jest-snapshot';
 
-export type Expect = typeof expect;
+export default (config: Config.GlobalConfig): Expect.JestExpect => {
+  const jestExpect = expect as Expect.JestExpect;
```

Comment:
Interface merging or casting? My thinking was: if it is possible without merging, then better to avoid another `declare module`. Merging is less explicit. Usually hiding in another file. Felt like code is more readable with type casting.

Reply:
agreed, if this can also work out for consumers outside of this module 🙂